kube/values.go

```diff
@@ -38,7 +38,12 @@ func MakeValues(settings ExportSettings) (helm.Node, error) {
 	}
 	env.Sort()
 
-	sizing := helm.NewMapping("HA", false)
+	sizing := helm.NewMapping(
+		"HA", false,
+		"memory", helm.NewMapping("slack", settings.MemLimitFactor,
```

Comment:
I don't like the term `slack` here; in my mind slack is something you add, not multiply. 😄

I would also prefer that we don't use different terms internally and externally, so it should be called `limit-factor` by default (or if we use a different user term, then the internal name should be updated to match).

And whatever we choose, we should also add a comment to it, explaining exactly what it does.

(and if you are feeling generous, add a comment to the `HA` setting as well).

Reply:
And by comment I mean a comment in the `values.yaml` file, not in the fissile source code. 😄

Reply:
sure.

Reply:
Addressed with commit d7d1dd7ccb049ac3889562f1a7daa511bc7b3160
(Note, using `limit_factor` because the dash chokes kube.)